src/Tribe/RSVP.php

```diff
@@ -1207,7 +1212,27 @@ public function front_end_tickets_form( $content ) {
 	 */
 	public function login_required() {
 		$requirements = (array) tribe_get_option( 'ticket-authentication-requirements', array() );
-		return in_array( 'event-tickets_rsvp', $requirements );
+		return in_array( 'event-tickets_rsvp', $requirements, true );
+	}
+
+	/**
+	 * Determine whether to use new RSVP views.
+	 *
+	 * @since TBD
+	 *
+	 * @return bool Whether to use new RSVP views.
+	 */
+	public function use_new_views() {
+		// Determine if ET was installed at version 4.12.2+.
+		$should_default_to_on = ! tribe_installed_before( 'Tribe__Tickets__Main', '4.12.2' );
+
+		return (
+			(
+				defined( 'TRIBE_TICKETS_RSVP_NEW_VIEWS' )
```

Comment:
I would love for us to use the structure from what we did on Views V2 for this one, where we check each one and return right after so one can overwrite the other, with also a filter for those using the option.

Reply:
Do you have a reference I can use for that?

Reply:
https://github.com/moderntribe/the-events-calendar/blob/master/src/functions/views/provider.php#L14